oojs.js

```diff
@@ -10,3 +10,27 @@
   // Have an artist property that is determined by some input.
 
 // Create an instance of the Album class.
+ class Playlist {
+   constructor (title, song){
+     this.title= title;
+     this.song  = song;
+     this.favorites = []
+   }
+
+   addSong (song){
+     this.favorites.push (song)
+
+   }
+
+ }
+ const tune = new Playlist ("Bad Romance", "Lady Gaga");
```

Comment:
A playlist's `songs` property should be able to contain multiple songs. This means that you need to pass in some sort of collection -- not a string. So creating a playlist should look something like...

```js
new Playlist("Super Bowl Playlist", ["Bad Romance", "Poker Face"])
```

Also remember that you defined the class so that the artist argument comes before the songs.